app/controllers/api/v1/search_controller.rb

```diff
@@ -516,10 +516,12 @@ def self.escape_terms_for_regex(term_list:)
       def self.generate_mongo_query_by_context(terms:, base_studies:, accessions:, query_context:)
         case query_context
         when :keyword
-          base_studies.any_of({:$text => {:$search => terms}}, {:accession.in => accessions})
+          author_match_study_ids = Author.where(:$text => {:$search => terms}).pluck(:study_id)
+          base_studies.any_of({:$text => {:$search => terms}}, {:accession.in => accessions}, {:id.in => author_match_study_ids})
```

Comment:
Blocker: for analytics on author search usefulness, and whether we hit the OKR, let's log to Mixpanel the studies returned by each query clause in `generate_mongo_query_by_context`.

Something like:
```suggestion
          by_text = {:$text => {:$search => terms}}
          by_accession = {:accession.in => accessions}
          by_author = {:id.in => author_match_study_ids}
          MetricsService.log('search-results', {
            numByAccession: by_accession.length
            numByText: by_text.length,
            numByAuthor: by_author.length
          }
          base_studies.any_of(by_text, by_accession, by_author)
```